warehouses/fase/ingest/clone_postgresql_table.py

```
@@ -0,0 +1,95 @@
+    
+    engine = create_engine(BASE_URI)
+    pipeline = dlt.pipeline(
+        pipeline_name=f"pg_to_iceberg_{args.schema}",
+        destination=pyiceberg,
+        dataset_name=args.schema,
+    )
+
+    try:
+        info = pipeline.run(postgres_dynamic_source(engine, args.schema, args.table))
+        print(info)
+    except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)